.env.example

```diff
@@ -1,6 +1,6 @@
 TWITTER_CREATOR="@medusajs"
 TWITTER_SITE="https://medusajs.com/"
 SITE_NAME="Next.js Commerce by Medusa"
-NEXT_PUBLIC_MEDUSA_BACKEND_API="http://localhost:9000/store"
+NEXT_PUBLIC_MEDUSA_BACKEND_API="http://localhost:9000"
```

Comment:
we can also default to this directly in the `medusa/index.ts` file?

Reply:
Yeah, we do in `lib/medusa/index.ts` on line 22!

Reply:
Sorry my bad!